### PR TITLE
feat: Adds info on adding teammates to the insights page

### LINKED ISSUE #202: Add  info on adding teammates to insights page
In the Insights feature page, we should include a portion that says something like:

> After you've created your insights page, share it with your team. 

1. Add an explanation why
2. Add directions
3. Add a link to this video: https://www.youtube.com/watch?v=L5ztLP1O7BY

### PR DESCRIPTION
**Pull request by Lymah123**
## Description

This PR adds the feature to the docs in the insights section. 

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

 Fixes #202 
[Add info on adding teammates to insights page](https://github.com/open-sauced/docs/issues/202) 

## Added to documentation?

- [ ] 📜 README.md
- [x] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Comment by Lymah123** [comment]
Hi @adiati98 , @BekahHW, and @CBID2. I will be waiting for your reviewsl

File: docs/features/insights.md
Comment (adiati98) [inline docs/features/insights.md:41]: For instruction to create a new Insight Page, please refer to the [docs here](https://docs.opensauced.pizza/maintainers/maintainers-guide-to-open-sauced/#creating-a-new-insight-page).

If what @BekahHW means here is to add direction about sharing Insights Page, we only need to add the instructions about how to share the page.

File: docs/features/insights.md
Comment (adiati98) [inline docs/features/insights.md:46]:
```suggestion
To learn more about inviting your team, check out [this tutorial video](https://www.youtube.com/watch?v=L5ztLP1O7BY).
```

File: docs/features/insights.md
Comment (adiati98) [inline docs/features/insights.md:36]: @BekahHW is the direction here just to add teammate to Insights Page or also creating an Insight Page?

```suggestion
## Adding Your Teammate to the Insights Page
```

**Review by adiati98 — CHANGES_REQUESTED** [review]
@Lymah123 I've left some reviews here. :)

@BekahHW we need your thoughts and confirmation on some things here.

TIA!

File: docs/features/insights.md
Comment (BekahHW) [reply docs/features/insights.md:41]: Right, we don't need instruction on how to create an insight page.

**Comment by Lymah123** [comment]
Hi @BekahHW and @adiati98, sorry for not being responsive for the past three days on your reviews. I have gone through your feedback. I have started working on them. Thanks for being patient with me. I will soon request for your reviews again .

**Comment by CBID2** [comment]
> Hi @BekahHW and @adiati98, sorry for not being responsive for the past three days on your reviews. I have gone through your feedback. I have started working on them. Thanks for being patient with me. I will soon request for your reviews again .

Hi @Lymah123! There’s some merge conflicts on your branch. Can you solve them please?

**Comment by Lymah123** [comment]
> > Hi @BekahHW and @adiati98, sorry for not being responsive for the past three days on your reviews. I have gone through your feedback. I have started working on them. Thanks for being patient with me. I will soon request for your reviews again .
> 
> Hi @Lymah123! There’s some merge conflicts on your branch. Can you solve them please?

Hi @CBID2 , I will resolve them. I just saw the conflicts on the PR today.

**Comment by Lymah123** [comment]
@CBID2 , I have resolved the conflicts.

File: docs/features/insights.md
Comment (CBID2) [inline docs/features/insights.md:29]: Remove space.

**Comment by CBID2** [comment]
> @CBID2 , I have resolved the conflicts.

Just gave some feedback [here](https://github.com/open-sauced/docs/pull/230#discussion_r1429007306) @Lymah123

File: docs/features/insights.md
Comment (Lymah123) [reply docs/features/insights.md:36]: Hi @BekahHW , @adiati98 and @CBID2 . Can I include this recording or I shouldn't?

Comment (adiati98) [reply docs/features/insights.md:36]: @Lymah123 I think this recording is only to show you what @BekahHW means. So we don't need to include this.

**Comment by adiati98** [comment]
Hi @Lymah123,
How things going here? Do you need any help?

**Comment by Lymah123** [comment]
> Hi @Lymah123, How things going here? Do you need any help?

Hi @adiati98, you can review it now. Thank you!

File: docs/features/insights.md
Comment (adiati98) [inline docs/features/insights.md:46]: "Explore" is different with "Insights". So, let's fix this.
And let's use ordered instead of unordered list. :)

```suggestion
### Method 1: Using the Share Button

This method gives view-only access to your Insights Page.

1. Click the "Insights" tab at the top bar.
2. Click the page that you want to share.
3. On the top right, click the "Share" button to copy the Insights Page URL to the clipboard.
4. Share the URL with your teammates.
```

File: docs/features/insights.md
Comment (adiati98) [inline docs/features/insights.md:53]:
```suggestion
### Method 2: Adding a Teammate

This method allows you to invite your teammates to your Insights Page and offers you options to permit them to be an admin, edit or view the page, or remove them.

1. Click the "Insights" tab at the top bar.
2. Click the page that you want to share.
3. Click the "Edit Page" button on the top right.
4. Scroll down to the "Add Team Members" section.
5. Enter the email address of the teammate you want to invite.
6. Click the "Send invite" button.
    
   Once you click the button, your teammate's email will be listed at the bottom.
7. To select and give permission to your teammate, click the arrow button on the right side of their email.
```

File: sidebars.js
Comment (adiati98) [inline sidebars.js:122]: I'm not sure what's going on here.
You changes doesn't require updating the `sidebars.js` and the changes here is causing error and terminated the docs locally.
Can you please update your branch before working on your changes?

**Review by adiati98 — CHANGES_REQUESTED** [review]
Hi @Lymah123,
Thanks for the changes! I gave you some feedback here. :)

Also, have you tested and run the docs locally?
Currently, there are unnecessary changes in the `sidebars.js` that are causing errors and shut the docs down locally, as shown in the screenshot below:

docs error

You might want to address this one first before working on the feedback. 😊

For future reference, you always want to run projects locally to test them and ensure they run properly.

File: docs/features/insights.md
Comment (adiati98) [inline docs/features/insights.md:35]:
```suggestion
### How to Share Insights Page
```

File: docs/features/insights.md
Comment (adiati98) [inline docs/features/insights.md:29]:
```suggestion
## Sharing Your Insights Page
```

File: docs/features/insights.md
Comment (adiati98) [inline docs/features/insights.md:33]:
```suggestion
Sharing this page with your team ensures that everyone is well-informed about the project's status and can collaborate more effectively. It fosters a collaborative environment where team members can align their efforts, make informed decisions, and collectively work toward the organization's success.
```

**Review by adiati98 — CHANGES_REQUESTED** [review]

**Comment by Lymah123** [comment]
Hi @adiati98, I have worked on the feedback. My branch is up to date. I ran my changes locally. It works perfectly. Besides, I didn't make changes to the sidebar section. What could be the issues?

**Comment by adiati98** [comment]
@Lymah123, if you see your PR here, there are 2 file changes. One of them is the sidebar. I'm not sure if you accidentally changed it?

It's pretty weird that it can run locally on your end after running `npm start` with the changes in the sidebar. 🤔

I'll take a look at this tomorrow as I'm not on my computer anymore.

In the meantime, can you give a screenshot or screen recording when it runs on your machine so I can take a look at it tomorrow? Thanks. 🙂

**Comment by Lymah123** [comment]
@adiati98 , I am not too sure. Do you support I should delete the repo, fork, then make the changes and submit a new PR?

**Comment by CBID2** [comment]
> @adiati98 , I am not too sure. Do you support I should delete the repo, fork, then make the changes and submit a new PR?

Hi @Lymah123. how about you try @adiati98's suggestion first? It might be a minor issue.

**Comment by Lymah123** [comment]
> > @adiati98 , I am not too sure. Do you support I should delete the repo, fork, then make the changes and submit a new PR?
> 
> Hi @Lymah123. how about you try @adiati98's suggestion first? It might be a minor issue.

Hi @CBID2 , okay. I will work on her suggestion. Thank you.

File: sidebars.js
Comment (adiati98) [inline sidebars.js:122]: @Lymah123 there are a couple things to look closer here:

- On line 117: This is your branch name. Perhaps you accidentaly typed it here.
- Line 118 - 121 is duplicate with line 123.

These are the reasons whenever we run `npm start`, the docs won't built locally.

You can go to the `sidebar.js` in the `main` branch and copy-paste the whole lines to fix this.

Here how this object suppose to look like:

https://github.com/open-sauced/docs/blob/0cc70484d6e945ac77930e755bdde3196a2db9fe/sidebars.js#L112-L118

Comment (Lymah123) [reply sidebars.js]: @adiati98 , you can check now. Thank you!

File: docs/features/insights.md
Comment (adiati98) [reply docs/features/insights.md:29]: @Lymah123 I see that you marked all newest suggestions as resolved, but there are no changes applied as per suggestions.

It's totally okay if you don't take the suggestions, but it would be great if you provide the reasoning behind your thoughts in each suggestion. That way, reviewers can understand your ideas. 🙂 

That said, would you please let me know why you don't apply them?

**Review by adiati98 — COMMENTED** [review]
Thanks for the changes, @Lymah123!

As you can now see in your PR, there is only one file that has changed. And that's what we want. 😊

I have a question for you here.
And FYI, I'm taking a few days off and will be back on 3rd January.
So, happy new year!

**Comment by Lymah123** [comment]
> Thanks for the changes, @Lymah123!
> 
> As you can now see in your PR, there is only one file that has changed. And that's what we want. 😊
> 
> I have a question for you here. And FYI, I'm taking a few days off and will be back on 3rd January. So, happy new year!

Awwnn! Happy New Year to you @adiati98! Always a pleasure having you review my PRs. Thanks for all the guidance you give. Looking forward to more of it in the coming year.

**Comment by Lymah123** [comment]
I see that you marked all newest suggestions as resolved, but there are no changes applied as per suggestions.

It's totally okay if you don't take the suggestions, but it would be great if you provide the reasoning behind your thoughts in each suggestion. That way, reviewers can understand your ideas. 🙂

That said, would you please let me know why you don't apply them?

Regards this @adiati98 , I made changes to all the suggestions mentioned so far. A reason I marked them as resolved. You can reference anyone I didn't make changes to and I would gladly do that. 

Thanks for always being patient with me @adiati98.

**Comment by adiati98** [comment]
@Lymah123 you can click the "Files changed" tab in your PR. Then, you can see the reviews that I've unresolved.

I'll take a look at them once I'm back. 👍🏽

**Comment by Lymah123** [comment]
> @Lymah123 you can click the "Files changed" tab in your PR. Then, you can see the reviews that I've unresolved.
> 
> I'll take a look at them once I'm back. 👍🏽

Alright. I will do that.  
Enjoy your day break @adiati98.

**Comment by Lymah123** [comment]
> @Lymah123 you can click the "Files changed" tab in your PR. Then, you can see the reviews that I've unresolved.
> 
> I'll take a look at them once I'm back. 👍🏽

Hi @adiati98, I have worked on unresolved suggestions.

**Review by adiati98 — APPROVED** [review]
I would love to have final review from @BekahHW here.
Bit it looks good to me. Thank you, @Lymah123! 🙌

**Review by BekahHW — APPROVED** [review]
Great job, @Lymah123! We're good to go!

**Pull request merged**
